elections/OpenSSF-TAC-Nominations-2026.md

```diff
@@ -0,0 +1,13 @@
+|Georg Kunz| Prounouns: He/Him Company: Ericsson [GitHub](https://github.com/gkunz) [LinkedIn](https://www.linkedin.com/in/georg-kunz/)|I am an a director in Ericsson's Open Source Program Office and a member of the OpenSSF technical community since 2022. During this time, I have participated in various TIs, committees and working groups, such as the Best Practices WG, the End Users WG, and the Global Cyber Policy Working Group. In the Best Practices WG, I have facilitated the creation of two SIGs: the C/C++ Compiler Options Hardening Guide and the Python Secure Coding Guide. I am also a regular speaker at OpenSSF Community Days as well as other Linux Foundation events, where I talk about the OpenSSF and its initiatives. Since end of 2024 I am co-chairing the Best Practices WG, and I am serving on the OpenSSF TAC since 2025. Having served on the TAC during 2025, I'd like to renew my commitment to the OpenSSF community for another term and work towards two goals: i) growing the technical community by strengthening our efforts to address the practical needs open source developers and project maintainers, and ii) fostering the development of methods and tools which help organizations - and OSPOs - to achieve compliance to upcoming cyber security regulation.|
+|Harald Fischer|Company: Ericsson [GitHub](https://github.com/fisehara) [LinkedIn](https://www.linkedin.com/in/harald-fischer-dus/)|My background is in embedded, medical device, and enterprise software, and I am now responsible for organisational and product cybersecurity at balena. balena has been developing and contributing to open source software from the beginning. This gives me direct experience in maturing open source projects to meet security and reliability demands of open, commercial and enterprise users. I want to contribute this experience to OpenSSF’s technical initiatives. The biggest challenge I see ahead for the community is navigating new regulations like the EU’s Cyber Resilience Act. As an active contributor, I believe my role is to help develop clear and easy guidance and practical frameworks. This will equip open source communities to provide needed and usable software for other initiatives and commercial users. I’d like to take the chance to strengthen my existing contributions to the OpenSSF on a broader level. I also want to learn from this community and collaborate with other leaders to improve the entire ecosystem.|
```

Comment:
As much as I'd love Harald to be affiliated with Ericsson, I am afraid he is with Balena 😉